packages/java/endpoint/src/main/java/dev/hilla/EndpointCodeGenerator.java

```diff
@@ -36,8 +37,12 @@ public class EndpointCodeGenerator {
     private final EndpointController endpointController;
     private final VaadinContext context;
     private Path buildDirectory;
+
     private ApplicationConfiguration configuration;
     private String nodeExecutable;
+    private Set<String> classesUsedInOpenApi = null;
+
+    private static EndpointCodeGenerator instance = null;
```

Comment:
Where's its value initialized?

Reply:
Feels like something has gone missing during the rebase

Reply:
Added